llama-index-integrations/tools/llama-index-tools-passio-nutrition-ai/examples/passio_nutrition_ai.ipynb

```diff
@@ -0,0 +1,108 @@
+      "========================\n",
+      "\n",
+      "The slice of pepperoni pizza you were eating contains approximately 2523.9 calories. It's important to note that this calorie count is for a 14\" pizza with pepperoni topping and regular crust. If you are looking for a healthier option, you may consider alternatives with fewer calories.\n"
```

Comment:
>The slice of pepperoni pizza you were eating contains approximately 2523.9 calories

I think we might need to change that example 😅 It's taken a whole (huge) pizza. Let's see if we can get an example with a good answer into the notebook

Reply:
Yes, I'll change it for sure. I realized it once I push it.